### PR TITLE
data: dev-phantom

### PR DESCRIPTION
**Pull request by dev-phantom**

### CODE REVIEW

File: data/devPhantom.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/dev-phantom",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/elimihele-god-s-favour-09341822b/"
```
Comment (priyankarpal) [inline data/devPhantom.json:18]: Add a comma

 ```suggestion
      "url": "https://www.linkedin.com/in/elimihele-god-s-favour-09341822b/",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
hi, @dev-phantom JSON file looks great, but you need to add your exact same GitHub username to the json file such as `data/dev-phantom.json` Please fix it. 

Screenshot from 2023-03-02 20-09-38

**Comment by dev-phantom** [comment]
@priyankarpal I've fixed this

**Comment by priyankarpal** [comment]
> @priyankarpal I've fixed this

You just added a new title. You need to change the json file name from `devPhantom.json` to `dev-phantom.json`. If you're facing any trouble you can check this official doc. https://docs.github.com/en/repositories/working-with-files/managing-files/renaming-a-file. Or ask to LinkFree discord https://discord.com/invite/jZQs6Wu

> Make sure that your file into the `data` folder.

**Comment by dev-phantom** [comment]
@priyankarpal I've fixed this

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by devvsakib — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**